data/tlg1701/tlg002/tlg1701.tlg002.1st1K-grc1.xml

```diff
@@ -0,0 +1,635 @@
+                </respStmt>
+                <respStmt>
+                    <persName>Bruce Robertson</persName>
+                    <resp>Technical Advisor</resp>
+                </respStmt>
+            </titleStmt>
+            <publicationStmt>
+                <authority>Harvard College Library</authority>
+                <idno type="filename">tlg1701.tlg002.1st1K-grc1.xml</idno>
+                <availability>
+                    <p>Available under a Creative Commons Attribution-ShareAlike 4.0 International
+                        License</p>
+                </availability>
+                <date>2016</date>
+                <publisher>Harvard College Library</publisher>
+                <pubPlace>United State</pubPlace>
+            </publicationStmt>
+            <sourceDesc>
+                <listBibl>
+                    <biblStruct>
+                        <monogr>
+                            <title>The Testament of Abraham</title>
+                            <editor>
+                                <persName>
+                                    <name>Montague Rhodes James</name>
+                                </persName>
+                            </editor>
+                            <author ref="urn:cts:greekLit:tlg1701" xml:lang="lat">Testamentum Abrahamae</author>
+                            <imprint>
+                                <publisher>Cambridge University Press</publisher>
+                                <pubPlace>Cambridge</pubPlace>
+                                <date>1892</date>
+                            </imprint>
+                            <biblScope unit="volume">2</biblScope>
+                        </monogr>
+                        <ref target="http://hdl.handle.net/2027/iau.31858011209131">Hathi Trust</ref>
+                    </biblStruct>
+                </listBibl>
+            </sourceDesc>
+        </fileDesc>
+        <encodingDesc>
+            <p>Text encoded in accordance with the latest EpiDoc standards (January 2014)</p>
+            <refsDecl n="CTS">
+                <cRefPattern matchPattern="(.+).(.+)" n="chapter"
+                    replacementPattern="#xpath(/tei:TEI/tei:text/tei:body/tei:div/tei:div[@n='$1']/tei:div[@n='$2'])"/>
+                <cRefPattern matchPattern="(.+)" n="book"
+                    replacementPattern="#xpath(/tei:TEI/tei:text/tei:body/tei:div/tei:div[@n='$1'])"
+                />
+            </refsDecl>
+        </encodingDesc>
+        <profileDesc>
+            <langUsage>
+                <language ident="grc">Greek</language>
+                <language ident="lat">Latin</language>
+            </langUsage>
+        </profileDesc>
+        <revisionDesc>
+            <change/>
+        </revisionDesc>
+    </teiHeader>
+    <text>
+        <body>
+            <div type="edition" xml:lang="grc" n="urn:cts:greekLit:tlg1701.tlg002.1st1K-grc1">
+                    <div type="textpart" subtype="book" n="2">
```

Comment:
Since the whole thing is considered only a single "book", I think we can delete this `<div>` and just have chapter-level divs. Make sure to change the refsDecl as well.